master/getting-started/kubernetes/upgrade/upgrade.md

```diff
@@ -30,10 +30,16 @@ Once you have upgraded {{site.prodname}}, you can [complete the upgrade](#comple
 1. Check the status of the upgrade as follows.
 
    ```
-   kubectl rollout status ds/calico-node -n kube-system
+   kubectl get pods -n kube-system | grep calico-node
    ```
    
-   For more information about this, refer to the [Kubernetes documentation](https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/#step-4-watching-the-rolling-update-status)
+   Verify that the status of all calico-node pods indicate `Running`.
```

Comment:
Probably want to note that `1/2` is expected prior to running `calico-upgrade complete`